packages/client/src/client.ts

```diff
@@ -131,7 +131,7 @@ class Client<FetcherRequestInit extends RequestInit = RequestInit> {
       throw new Error(`Unable to get available Countries List: ${response.statusText}`);
     }
 
-    return response.json();
+    return response.json() as Promise<unknown>;
```

Comment:
🍹 Using `as` can slow down typescript compilation time. If you don't want to use `as` here, you might try an explicit return type instead. `async fetchAvailableCountries(): Promise<unknown> {}`